src/sage/calculus/transforms/dft.py

```diff
@@ -412,11 +423,11 @@ def dst(self):
         EXAMPLES::
 
             sage: J = list(range(5))
-            sage: I = CC.0; pi = CC(pi)
-            sage: A = [exp(-2*pi*i*I/5) for i in J]
-            sage: s = IndexedSequence(A,J)
+            sage: I = CC.0; pi = CC(pi)                                                 # optional - sage.symbolic
```

Comment:
This works with `math.pi` too

Reply:
also `CC.pi()` avoids going through symbolic constants; not sure what's better.